### PR TITLE
Disabling NSJSON edge case test on iOS 11

### PR DESCRIPTION
**Pull request by schmidt-sebastian**
The "extremeDouble" tests for the iOS RTDB SDK rely on the fact that NSJSONSerialization can't correctly handle tiny double values. This has been fixed in iOS 11 and the tests fail.

I don't believe the tests add value in iOS 11 and hence I disabled them.

### CODE REVIEW

File: Example/Database/Tests/Unit/FLevelDBStorageEngineTests.m [hunk 1]
```diff
@@ -426,6 +426,13 @@ - (void)testHugeLeafNodeThenDeeperSet {
 // Well this is awkward, but NSJSONSerialization fails to deserialize JSON with tiny/huge doubles
 // It is kind of bad we raise "invalid" data, but at least we don't crash *trollface*
 - (void)testExtremeDoublesAsServerCache {
+#ifdef TARGET_OS_IPHONE
```
Comment (paulb777) [inline Example/Database/Tests/Unit/FLevelDBStorageEngineTests.m]: We're generally testing IOS with `#if TARGET_OS_IOS`

Comment (schmidt-sebastian) [inline Example/Database/Tests/Unit/FLevelDBStorageEngineTests.m]: Done.

**Comment by paulb777** [comment]
Fixes #131
